### PR TITLE
Add Sign in with Apple federated identity docs for Android

### PR DESCRIPTION
**Pull request by palpatim**
*Issue #, if available:*

*Description of changes:*
Add documentation for Sign in with Apple via Amazon Cognito Identity Pools.

**Note:** The code snippet has not yet been tested and should be verified before merging.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/sdk/auth/fragments/android/federated-identities.md [hunk 1]
```diff
@@ -150,6 +150,46 @@ You can now [configure Google in your mobile app](#google-login-in-your-mobile-a
...
+
+2. **Configuring Sign in with Apple as an authentication provider in your Amazon Cognito Identity Pool**
+
+    Once you have configured your application to use Sign in with Apple, paste your app's **Bundle Identifier** into the **Apple Services ID** field of your Amazon Cognito Identity Pool. The Bundle Identifier can be found in the **Certificates, IDs & Profiles** section of your Apple Account page.
```
Comment (brene) [inline docs/sdk/auth/fragments/android/federated-identities.md:163]: Can we have a link to the Apple account page? I literally tried googling it and cannot find anything resembling what is described here.

Comment (palpatim) [reply docs/sdk/auth/fragments/android/federated-identities.md]: Added link

Comment (brene) [inline docs/sdk/auth/fragments/android/federated-identities.md]: Link to Amazon Cognito console: https://console.aws.amazon.com/cognito/home

Comment (palpatim) [reply docs/sdk/auth/fragments/android/federated-identities.md]: Added

**Pull request merged**